### PR TITLE
Stripe api setup

### LINKED ISSUE #40: Setup: Stripe API
<https://stripe.com/docs/api/customers/create>

<https://stripe.com/docs/payments/checkout/subscriptions>

Set up a dev account and use the appropriate API keys set up for the app.

### PR DESCRIPTION
**Pull request by panwaramita**
### What this PR does (required):
- Set Up a developer account on stripe.com.
- Get the secret key and add it to the .env file.
- create a controller and route for:
- creating customer
- update customer
- retrieve customer
- retrieve customer
- create a checkout session

close #40

### CODE REVIEW

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
Hey @panwaramita, use the correct linters setup and format your code. Let's get this done to tidy up our code.
Happy coding!

File: server/controllers/stripe.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+// @route POST /stripe/customer/create
+// @desc create new customer
+// @access Public
+export async function createCustomer(req, res) {
+    const customer = await stripe.customers.create({
+        description: req.body.description,
+    });
+    res.status(200).json({
+        success: {
+            customer: customer,
```
[the remark this answers is not in the document]
Comment (panwaramita) [reply server/controllers/stripe.js:13]: Done the changes

**Review by ezeilo-su — COMMENTED** [review]
Thank you for the requested changes.

File: server/app.js [hunk 2]
```diff
@@ -12,8 +12,13 @@ const logger = require("morgan");
 const authRouter = require("./routes/auth");
 const userRouter = require("./routes/user");
 const profileRouter = require('./routes/profile');
+<<<<<<< HEAD
+const stripeRouter = require('./routes/stripe');
+=======
+const stripeRouter =require('./routes/stripe');
 const imageRouter = require('./routes/image');
 
+>>>>>>> 4d697bc7232c3803f506e6ae2286e1cd28974508
```
Comment (rajivtitus) [inline server/app.js:21]: Let's remove these commit hashes

Ditto throughout

**Review by rajivtitus — APPROVED** [review]
Good start with the routes. We'll have to refactor them to fit our use cases, we can do that in the subsequent payment related tickets 👍🏽  

Just a comment to address before merging

**Pull request merged**